Engine/Component/ComponentTransform.cpp

```diff
@@ -192,4 +212,10 @@ void ComponentTransform::ChangeLocalSpace(const float4x4 &new_local_space)
 	model_matrix = new_local_space.Inverted() * global_model_matrix;
 	model_matrix.Decompose(translation, rotation, scale);
 	
-}
\ No newline at end of file
+}
+
+Component* ComponentTransform::Clone(bool original_prefab) const
+{
+
+	return nullptr;
+}
```

Comment:
Same as in ComponentAABB, this cannot be just in the header file?

Reply:
I just implemented them. But probably they are not going to be use